### PR TITLE
Updated logo url on dashboard

### PR DESCRIPTION
**Pull request by RamanaReddy8801**
# Summary
**Ticket**: https://issues.newrelic.com/browse/NR-102766
**Description**: Updated the python logo url on dashboard.

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Review by pkosanam — APPROVED** [review]
looks good

File: dashboards/lampy/lampy.json [hunk 1]
```diff
@@ -485,7 +485,7 @@
             "id": "viz.markdown"
           },
           "rawConfiguration": {
-            "text": "![Python](https://s3.dualstack.us-east-2.amazonaws.com/pythondotorg-assets/media/community/logos/python-logo-only.png)"
+            "text": "![Python](https://raw.githubusercontent.com/newrelic/newrelic-quickstarts/ec054ad274f7ad248184911c445f2c711917076e/quickstarts/python/python/logo.svg)"
```
Comment (mdumpati) [inline dashboards/lampy/lampy.json:488]: @RamanaReddy8801 This logo is not showing full image so replace this?

**Review by mdumpati — COMMENTED** [review]
@RamanaReddy8801 Please check my comments

File: dashboards/lampy/lampy.json (hunk 1 above)
Comment (RamanaReddy8801) [reply dashboards/lampy/lampy.json:488]: In the dashboard I am able to see full logo with this image, please have a look below:

Screenshot 2023-05-31 at 10 20 28 PM
Screenshot 2023-05-31 at 10 20 09 PM

**Review by mdumpati — APPROVED** [review]
LGTM

**Pull request merged**